src/modules/fancyzones/lib/fancyzones.rc

```diff
@@ -8,6 +8,7 @@ BEGIN
     IDS_SETTING_DESCRIPTION                                    "Create window layouts to help make multi-tasking easy"
     IDS_SETTING_DESCRIPTION_SHIFTDRAG                          "On: Hold Shift key or any non-primary mouse button to enable zones while dragging"
     IDS_SETTING_DESCRIPTION_OVERRIDE_SNAP_HOTKEYS              "Override Windows Snap hotkeys (win+arrow) to move windows between zones"
+    IDS_SETTING_DESCRIPTION_MOVE_WINDOW_ACROSS_MONITORS        "Move windows between zones across monitors"
```

Comment:
This may need more explanation: "Move windows between zones across monitors"

"Move windows between zones across all monitors when snapping with win+arrow"

Reply:
I agree that this one in PR is not the best one :) I'm open for proposals. It's way more clear how you decribed it

Reply:
Done.